package.json

```diff
@@ -20,6 +20,9 @@
     "updateOncoKbAPI": "npm run fetchOncoKbAPI && npm run buildOncoKbAPI",
     "fetchOncoKbAPI": "curl http://oncokb.org/api/v1/v2/api-docs > src/shared/api/generated/OncoKbAPI-docs.json",
     "buildOncoKbAPI": "node scripts/generate-api.js src/shared/api/generated OncoKbAPI",
+    "updateGenomeNexusAPI": "npm run fetchGenomeNexusAPI && npm run buildGenomeNexusAPI",
+    "fetchGenomeNexusAPI": "curl http://localhost:38080/v2/api-docs?group=variant_annotation > src/shared/api/generated/GenomeNexusAPI-docs.json",
```

Comment:
@athum could you point this to the heroku instance https://genome-nexus-mutation-assessor.herokuapp.com